docs/content/grafana/README.md

```diff
@@ -21,12 +40,54 @@ To add your dashboards to Grafana, you will need to create a `GrafanaDashboard`
 ## Using the Dashboards
 
 Grafana dashboards can easily be exported and imported, either from the UI or from the HTTP API.
-Dashboards are exported in Grafana JSON format, and contain everything you need (layout, variables, styles, data sources, queries, etc) to import the dashboard at a later time.
+Dashboards are exported in Grafana `JSON` format, and contain everything you need (layout, variables, styles, data sources, queries, etc) to import the dashboard at a later time.
 
-You can import these dashboards in Grafana either by pasting the dashboard JSON text directly into the text area or directly uploading the JSON file. Make sure to add/connect your [data sources][8] to Grafana and pick what data source you want the dashboard to use.
+You can import these dashboards in Grafana either by pasting the dashboard `JSON` text directly into the text area or directly uploading the `JSON` file. Make sure to add/connect your [data sources][8] (official Grafana data source documentation) to Grafana and pick what data source you want the dashboard to use.
 
 For more details on importing/exporting dashboards, you can refer to the Grafana documentation [here][9].
 
+## Creating Your First Dashboard
+
+If you've gained access to the Operate First Grafana instance, you should see the dashboard pictured above.
+
+On the left side bar there is a "+" button. Hover over it and click "Dashboard". This will create a new dashboard for you, where Grafana should redirect you immediately to.
```

Comment:
>  This will create a new dashboard for you, where Grafana should redirect you immediately to.

change to: 

>  This will create and redirect you to a new Dashboard